test/cluster/mzcompose.py

```diff
@@ -743,7 +742,9 @@ def workflow_pg_snapshot_resumption(c: Composition) -> None:
         # Start postgres for the pg source
         Postgres(),
         Testdrive(no_reset=True),
-        Storaged(environment=["FAILPOINTS=pg_snapshot_failure=return"]),
+        Clusterd(
+            name="storaged", environment=["FAILPOINTS=pg_snapshot_failure=return"]
```

Comment:
might want a todo to clean this up!

Reply:
Tests have all been adjusted to avoid using the terms "computed" or "storaged".